util/src/get_root_ksks.rs

```diff
@@ -58,6 +58,7 @@ pub fn main() {
                     | Algorithm::RSASHA512 => "rsa",
                     Algorithm::ECDSAP256SHA256 | Algorithm::ECDSAP384SHA384 => "ecdsa",
                     Algorithm::ED25519 => "ed25519",
+                    Algorithm::Unknown(_) => "unknown",
```

Comment:
can you put a TODO in here that we might want to make this something like `unknown-{value}` so that that the custom value isn't lost?

Reply:
I considered that but didn't want to alter the static lifetime on the returned string. If you're cool with the extra alloc to build it I can change this.

Reply:
Yeah, it's a minor utility... not worth doing in this change. I think it's converted to a `String` just a little later when it's added to the Path.